### PR TITLE
Dev paradata collection date

### PR DESCRIPTION
**Pull request by alexisszmundy**
- Extraction from paradatas (event with "agree-sending-modal-button-orchestrator-collect") as IdParadataObject
- Add paradata test data

### CODE REVIEW

File: kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/Constants.java [hunk 1]
```diff
@@ -64,6 +64,8 @@ private Constants() {}
 	public static final String NUMBER_ORCHESTRATORS_NAME = "NB_ORCHESTRATORS";
 	public static final String NUMBER_SESSIONS_NAME = "NB_SESSIONS";
 	public static final String PARADATA_VARIABLES_PREFIX = "CHANGES_";
+	public static final String PARADATA_TIMESTAMP_SUFFIX = "_LONG";
+	public static final String COLLECTION_DATE_NAME = "DATE_COLLECTE";
```
Comment (alicela) [inline kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/Constants.java:68]: Change "COLLECTION" in "SURVEY_VALIDATION"

File: kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/paradata/ParaDataUE.java [hunk 2]
```diff
@@ -28,6 +28,8 @@ public class ParaDataUE {
 	private List<Session> sessions = new ArrayList<>();
 	@Getter
 	private List<Orchestrator> orchestrators = new ArrayList<>();
+	@Getter
+	private long collectionDateTimeStamp;
```
Comment (alicela) [inline kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/paradata/ParaDataUE.java:32]: same idea

File: kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/paradata/ParaDataUE.java [hunk 3]
```diff
@@ -209,4 +211,17 @@ private Session changeCurrentSession(Session session, Event currentEvent, Event
 		return session;
 	}
 
+	/**
+	 * Gets the validation timestamp from events and put it into the collectionDateTimeStamp
+	 * @param validationParadataObjectId idParadataObject considered as validation event
+	 */
+	public void setCollectionDateTimeStamp(String validationParadataObjectId) {
```
Comment (alicela) [inline kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/paradata/ParaDataUE.java:218]: same idea

File: kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/paradata/ParadataParser.java [hunk 4]
```diff
@@ -51,6 +54,7 @@ public void parseParadata(Paradata paradata, SurveyRawData surveyRawData) throws
 					paraDataUE.setFilepath(fileParaDataPath);
 					parseParadataUE(paraDataUE, surveyRawData);
 					paraDataUE.sortEvents();
+					paraDataUE.setSurveyValidationDateTimeStamp("agree-sending-modal-button-orchestrator-collect");
```
Comment (loichenninger) [inline kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/extradata/paradata/ParadataParser.java:57]: Maybe define this String as the constant

**Pull request merged**